services/ui-src/src/page/section/TemporaryExtensionSection.tsx

```diff
@@ -124,19 +126,24 @@ export const TemporaryExtensionSection: FC<{
     <section id="temp-ext-base" className="read-only-submission ">
       <div className="mini-dashboard-title">
         <h2>Temporary Extensions</h2>
-        <Link<FormLocationState>
-          to={{
-            pathname: ONEMAC_ROUTES.TEMPORARY_EXTENSION,
-            state: {
-              parentId: detail.componentId,
-              parentType: detail.componentType,
-            },
-          }}
-        >
-          <Button id="new-temp-ext-button" variation="primary">
-            Request Extension
-          </Button>
-        </Link>
+        {userRoleObj.canAccessForms &&
+          pageConfig.actionsByStatus[detail.currentStatus]?.includes(
+            Workflow.PACKAGE_ACTION.REQUEST_TEMPORARY_EXTENSION
+          ) && (
```

Comment:
Found a bug that allowed TE to be submitted via the TE mini dashboard, therefore tightened up the logic to allow the button under same circumstances as the link in the details section

Reply:
Very nice find - I would be reluctant to call it a bug, since I'm pretty sure there are no ACs around the button on the mini-dashboard being there or not there based on package status or user permissions.... but I agree that it is the right thing to do (might want to bring it up to product though, or mark it in the Figma?)

Reply:
True, more of a possible requirements miss than a bug. I do plan on bringing this up to Ryan/Micha/Jaime.